webpack/utils/helpers.js

```diff
@@ -81,12 +81,15 @@ export const sendErrorNotifications = messages => (dispatch) => {
 export const apiError = (actionType, result, additionalData = {}) => (dispatch) => {
   const messages = getResponseErrorMsgs(result.response);
 
-  const dataExtenstion = {
-    messages,
-    ...additionalData,
-  };
+  // If no actionType passed in, only create notification and skip dispatching action
+  if (actionType) {
+    const dataExtenstion = {
```

Comment:
```suggestion
 let dataExtension;
  if (actionType) {
    dataExtension = {
```
If no actionType is passed in, `dataExtension` will never be defined, which will cause a `ReferenceError: dataExtension is not defined`